packages/brick_sqlite/lib/src/helpers/query_sql_transformer.dart

```diff
@@ -347,6 +351,17 @@ class WhereColumnFragment {
 
     return ' $matcher ${wherePrepared.join(' $matcher ')}';
   }
+
+  String _generateInList() {
+    final value = condition.value;
+    if (value is! Iterable || value.isEmpty) {
+      // Always false condition, use matcher for consistency
+      return ' $matcher 1=0';
```

Comment:
Why should this always be false? That seems unexpected behavior
```suggestion
      return '';
```

Reply:
This is intentional and correct behavior. Returning ' $matcher 1=0' ensures that queries with an empty IN list do not cause SQL errors and return no results, which matches the expected logic.